src/main/java/org/broadinstitute/hellbender/tools/walkers/genotyper/GenotypingEngine.java

```diff
@@ -164,6 +166,12 @@ public VariantContext calculateGenotypes(final VariantContext vc, final Genotype
             reducedVC = new VariantContextBuilder(vc).alleles(allelesToKeep).genotypes(reducedGenotypes).make();
         }
 
+        final long maxPLLength = GenotypeLikelihoods.numLikelihoods(reducedVC.getNAlleles(), reducedVC.getMaxPloidy(defaultPloidy)) * reducedVC.getNSamples();
+
+        if(maxPLLength >= TOO_LONG_PL)
+        {
+            logger.warn("Total length of all PL arrays for this VC is likely to reach " + maxPLLength + ", so processing may take a long time.");
```

Comment:
It would be good to include the following things in this message: 
1.  variant position
2. the number of alleles at the site
3. the ploidy
4. the predicted number of pls

Reply:
Good point.  Added that extra info